$OpenBSD$

Both BSD m4 and GNU m4 support -g. GNU m4 supports --gnu, but BSD m4 does
not.  Therefore, always use -g. Note that the m4 implementation autogunk is
detecting here is not the one that will be used at runtime. See the other
patch.

--- configure.orig	Fri Aug 23 15:10:06 2013
+++ configure	Fri Aug 23 15:14:27 2013
@@ -8774,11 +8774,7 @@ else
 fi
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_cv_prog_gnu_m4_gnu" >&5
 $as_echo "$ac_cv_prog_gnu_m4_gnu" >&6; }
-  if test "$ac_cv_prog_gnu_m4_gnu" = yes; then
-    M4_GNU=--gnu
-  else
-    M4_GNU=
-  fi
+  M4_GNU=-g
 
   if test x$ac_had_posixly_correct = xyes; then
     POSIXLY_CORRECT=:
